influxql/query_executor.go

```diff
@@ -66,6 +66,9 @@ type ExecutionOptions struct {
 
 	// Node to execute on.
 	NodeID uint64
+
+	// Do not log the normalized statement for this query.
+	Quiet bool
```

Comment:
The two are separate objects so that would also disable all query logging for the query executor rather than just disabling logging for continuous queries.